installation/template/js/remove.js

```diff
@@ -36,37 +36,68 @@ if (document.getElementById('removeInstallationFolder')) {
 			e.preventDefault();
 			let confirm = window.confirm(Joomla.Text._('INSTL_REMOVE_INST_FOLDER').replace('%s', 'installation'));
 			if (confirm) {
-				Joomla.request({
-					method: "POST",
-					url: Joomla.installationBaseUrl + '?task=installation.removeFolder&format=json',
-					perform: true,
-					token: true,
-					headers: { 'Content-Type': 'application/x-www-form-urlencoded' },
-					onSuccess: function (response) {
+			    Joomla.deleteJoomlaInstallationDirectory();
+			}
+		});
+}
+
+const completeInstallationOptions = document.querySelectorAll('.complete-installation');
+
+completeInstallationOptions.forEach(function(item) {
+    item.addEventListener('click', function (e) {
+        // Evil voodoo. Once a button is clicked ensure they don't spam click it...
```

Comment:
```suggestion
        // Once a button is clicked ensure they can't click it again...
```

Lets keep it professional ;)